src/Database/Driver/Postgres.php

```diff
@@ -92,6 +97,24 @@ public function connect(): void
             $dsn = "pgsql:dbname={$config['database']}";
         }
 
+        if ($this->_config['ssl']) {
+            if ($this->_config['ssl_mode']) {
+                $dsn .= ';sslmode=' . $this->_config['ssl_mode'];
+            } else {
+                $dsn .= ';sslmode=' . 'allow';
+            }
+
+            if ($this->_config['ssl_key']) {
+                $dsn .= ';sslkey=' . $this->_config['ssl_key'];
+            }
+            if ($this->_config['ssl_crt']) {
+                $dsn .= ';sslcert=' . $this->_config['ssl_crt'];
```

Comment:
```suggestion
                $dsn .= ';sslcert=' . $this->_config['ssl_cert'];
```

We use `ssl_cert` for mysql connections, it would be good to have consistent conventions.